plottable.d.ts

```diff
@@ -3680,15 +3692,26 @@ declare module Plottable {
 
 declare module Plottable {
     class Interaction {
+        protected _componentAttachedTo: Component;
+        protected _anchor(component: Component): void;
+        protected _unanchor(): void;
         /**
-         * It maintains a 'hitBox' which is where all event listeners are
-         * attached. Due to cross- browser weirdness, the hitbox needs to be an
-         * opaque but invisible rectangle.  TODO: We should give the interaction
-         * "foreground" and "background" elements where it can draw things,
-         * e.g. crosshairs.
+         * Attaches current interaction to a Component. If the interaction was already
```

Comment:
I don't this the word `current` here is nice in the doc... I would remove the word completely or replace it with `this`